rpc/src/v1/types/confirmations.rs

```diff
@@ -324,7 +329,8 @@ mod tests {
 	fn should_deserialize_modification() {
 		// given
 		let s1 = r#"{
-			"gasPrice":"0xba43b7400"
+			"gasPrice":"0xba43b7400",
+			"minBlock":42
```

Comment:
I think we encode block numbers as hex almost everywhere, would be good to keep consistency.